amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -90,49 +93,48 @@ def test_getAGBarcodeDetails(self):
             'barcode': '000001047',
             'status': 'Received',
             'ag_kit_id': 'd8592c74-7e35-2135-e040-8a80115d6401',
-            'name': 'REMOVED',
-            'participant_name': 'REMOVED-0',
-            'email': 'REMOVED',
             'site_sampled': 'Stool',
             'environment_sampled': None,
             'sample_date': datetime.date(2013, 3, 28),
             'sample_time': datetime.time(23, 25),
-            'notes': 'REMOVED',
             'overloaded': None,
             'withdrawn': None,
             'other': None,
             'moldy': None,
             'refunded': None,
             'ag_kit_barcode_id': 'd8592c74-7e36-2135-e040-8a80115d6401',
             'date_of_last_email': None,
-            'other_text': 'REMOVED'
         }
-        self.assertEqual(obs, exp)
+        # only look at those fields, that are not subject to scrubbing
+        self.assertEqual({k: obs[k] for k in exp}, exp)
 
     def test_getAGKitDetails(self):
         # test non-existant kit
         with self.assertRaises(ValueError):
             self.ag_data.getAGKitDetails('IDONTEXI5T')
 
         # test existing AG kit
-        obs = self.ag_data.getAGKitDetails('tst_ODmhG')
+        obs = self.ag_data.getAGKitDetails(
+            self.ag_data.ut_get_supplied_kit_id(
+                'd8592c74-84ba-2135-e040-8a80115d6401'))
+        # deleted password, since it is not stable along DB versions
         exp = {'ag_kit_id': 'd8592c74-84bb-2135-e040-8a80115d6401',
-               'supplied_kit_id': 'tst_ODmhG',
-               'swabs_per_kit': 1,
+               'supplied_kit_id':
+               self.ag_data.ut_get_supplied_kit_id(
+                'd8592c74-84ba-2135-e040-8a80115d6401'),
+               'swabs_per_kit': 1L,
                'verification_email_sent': 'n',
-               'kit_verification_code': 'f4UjhV4B',
-               'kit_password': '$2a$12$LiakUCHOpAMvEp9Wxehw5OIlD/TIIP0Bs3blw18'
```

Comment:
agreed. I put password back in. And it should be the only place where we compare password string for unit tests